addons/website/views/snippets/s_banner_categories.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_banner_categories" name="Banner Categories">
+    <section
+        data-name="Banner Categories"
+        class="s_banner_categories o_cc o_cc3 oe_img_bg o_bg_img_center pb56"
+        data-oe-shape-data="{'shape':'web_editor/Connections/20','colors':{'c5':'rgb(255, 255, 255)'},'showOnMobile':false}"
+        style="background-image: url('/web/image/website.s_banner_categories_default_image_1'); background-position: 50% 75%;"
+        >
+        <div class="o_we_bg_filter bg-black-50"/>
+        <div
+            class="o_we_shape o_web_editor_Connections_20"
+            style="background-image: url('/web_editor/shape/web_editor/Connections/20.svg?c5=rgb%28255%2C255%2C255%29');"
+            />
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="11" style="gap: 15px;">
```

Comment:
Nitpick: usually we use spacing values based on a 8px scale to keep consistency (see our default spacers)
You can check other places where you had to hardcode spacing values 👍 

([interesting article](https://medium.com/@mertyagci/the-power-of-the-8pt-grid-system-in-design-1c9dbc683ad8))